### PR TITLE
Fixes how pulpcore-manager command is discovered

### PR DESCRIPTION
**Pull request by dkliban**
[noissue]

### CODE REVIEW

**Review by bmbouter — APPROVED** [review]
Ha excellent

File: pulp_smash/utils.py [hunk 1]
```diff
@@ -159,8 +159,8 @@ def execute_pulpcore_python(cli_client, command):
         bin_dir = "/usr/local/bin"
         for line in ps_output:
             if "pulpcore" in line:
-                if "bin/python" in line.split()[4]:
-                    bin_dir = line.split()[4].rsplit("/", maxsplit=1)[0]
+                if "bin/python" in line.split()[5]:
+                    bin_dir = line.split()[5].rsplit("/", maxsplit=1)[0]
```
Comment (mdellweg) [inline pulp_smash/utils.py:163]: Assuming this has worked for someone before, the format of ps must have chaged. Can we use something more stable? Maybe `ps ax -o command`?

**Review by mdellweg — CHANGES_REQUESTED** [review]

File: pulp_smash/utils.py (hunk 1 above)
Comment (dkliban) [reply pulp_smash/utils.py:163]: This command was never run inside the container before. It was only run inside our developer VMs. This change continues to work inside a VM and also inside the centos based container.

**Comment by dkliban** [comment]
@mdellweg I am merging this as it is now so I can stop requiring this PR everwhere. Let me know if you want me to change it though.

**Pull request merged**